src/main/java/frc/robot/constants/AutoConstants.java

```diff
@@ -1,17 +1,83 @@
 package frc.robot.constants;
 
 import java.util.HashMap;
+import java.util.Map;
 import com.pathplanner.lib.PathConstraints;
 import com.pathplanner.lib.PathPlanner;
 import com.pathplanner.lib.PathPlannerTrajectory;
 import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.RunCommand;
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import edu.wpi.first.wpilibj2.command.WaitCommand;
+import frc.robot.commands.ScoreCone;
+import frc.robot.constants.ArmConstants.ArmState;
+import frc.robot.subsystems.ArmControlSubsystem;
+import frc.robot.subsystems.IntakeSubsystem;
+
 
 public class AutoConstants {
 
-    public static PathConstraints defaultSpeedConstraints = new PathConstraints(4, 2);
+    private AutoConstants(){
+        throw new IllegalStateException("Constants Class");
+    }
+
+    public static final PathConstraints kDefaultSpeedConstraints = new PathConstraints(3, .75);
+
+    public static final PathPlannerTrajectory kStation2PieceBlue = PathPlanner.loadPath("Station2PieceBlue", kDefaultSpeedConstraints);
+    public static final PathPlannerTrajectory kStation2PieceRed = PathPlanner.loadPath("Station2PieceRed", kDefaultSpeedConstraints);
+
+    public static final PathPlannerTrajectory kMoveOneMeter = PathPlanner.loadPath("MoveOneMeter", kDefaultSpeedConstraints);
+
+    public static Map<String, Command> emptyMap = new HashMap<>();
+    public static Map<String, Command> eventMap = new HashMap<>();
+
```

Comment:
Should this be using a different naming convention?

Reply:
You are right, it should follow the all caps convention. Done.